src/Finder/ModuleFileFinder.php

```diff
@@ -46,4 +57,45 @@ protected function _filter(SplFileInfo $fileInfo)
     {
         return $fileInfo->isReadable();
```

Comment:
Perhaps, it is better to let the locator [throw](https://github.com/RebelCodecom/module-locator/blob/develop/src/FileLocator.php#L149) with helpful info, instead of just silently ignoring the file to leave the user wondering why their config is not having an effect?